### DISCUSSION TITLE
TL-WN725N Not working on monterey 12.71

### DISCUSSION THREAD

**Discussion by DERMOIIN** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#336] [Q&A]
I already installed the thing from github, the installation is completed succesfully but whenever i try to connect the internet using the apps i installed, its just wont connect and in system preferences>network, my usb seems to be not connected to PC. However the USB adapter is showing green light, mean that the USB is active. Any solution?
![Screen Shot 2024-03-12 at 14 11 33](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/110816806/0d0a47e3-f69d-4c53-acf7-21aef4c6edf7)

**Comment by chris1111** [comment]
TL-WN725N work with V2 or V3
Not V1

**Reply by DERMOIIN** [reply]
thanks for the clarification, i couldnt find what version my usb was, because i stole it from desktop PC on my workshop

**Reply by DERMOIIN** [reply]
Hi again, i already buy the tenda W311-MI but now its showup nothing, and still the same

**Reply by DERMOIIN** [reply]
![17104225582528686881141111991133](https://github.com/chris1111/Wireless-USB-OC-Big-Sur-Adapter/assets/110816806/17003988-00f3-4f8b-b574-e650572c5b11)
Iam using this, the USB adapter working fine in Windows 11 but not on my hackintosh Monterey, there is something wrong on the software
